README

```diff
@@ -18,6 +18,12 @@ See https://wiki.gnome.org/Projects/SessionManagement/
 Installation
 ============
 
+In order to build this program from the source code you need git installed and
+ a working MATE environment version 1.2, with the development tools installed
+properly.
```

Comment:
1.2 ???
Where is this from? 
Damned, i over read that in mate-terminal PR.....

Reply:
Damned too, copy/paste error... I need to modify.

Reply:
Updated

Reply:
A tip, `git log -p` will show you the commit in terminal after commiting localy.

Reply:
Thanks, this could help.